build.sbt

```diff
@@ -137,6 +138,7 @@ lazy val core = projectMatrix
     isMimaEnabled := true,
     allowedNamespaces := Seq(
       "smithy.api",
+      "smithy.test",
```

Comment:
possible question: should `smithy.test` traits be included in smithy4s-core? Looks like they will.

Reply:
oh, I did not think of that, that's a good point. I'll try and move the codegen for this namespace in the complaince-test module instead

Reply:
did that! good catch